DESCRIPTION

```diff
@@ -92,12 +92,13 @@ Suggests:
     ggplot2,
     reactlog (>= 1.0.0),
     magrittr,
-    shinytest,
+    shinytest (> 1.3.1),
```

Comment:
This doesn't do anything in Suggests. We should add a version check in `shinyAppTemplate()`.